src/main/resources/knowledge_environment.graphqls

```diff
@@ -12,6 +12,7 @@ type Query {
   getSummaryData: [GeneDatasetInformation]
   getDataTypeInformationByParticipant(redcapId: String!): ParticipantDataTypeSummary
   participantSummaryDataset(redcapId: String): ParticipantSummaryDataset 
+  participantClinicalDataset(redcapId: String): ParticipantSummaryDataset

```

Comment:
Ideally this would be it's own separate type, unfortunately I was having issues with getting the autowire within spring to allow me to do so. This creates a slight oddity with the type below